behave_reportportal/behave_agent.py

```diff
@@ -401,6 +405,26 @@ def _get_attributes_from_tags(tags):
 
         return result
 
+    @staticmethod
+    def _test_case_id(scenario):
+        if scenario.tags:
+            tc_tag = next(
+                iter(
+                    [t for t in scenario.tags if t.startswith("test_case_id(")]
+                ),
+                None,
+            )
+            if not tc_tag:
+                return
+            start = tc_tag.find("(")
+            end = tc_tag.find(")")
+            if start == -1 or end == -1:
+                return
+            tc_id = tc_tag[start + 1 : end]
+            if not tc_id:
```

Comment:
Maybe just `return tc_id`?

Reply:
It's necessary to send None (not empty string) to R, because in such case it will build testCaseId based on code_ref